var/spack/repos/builtin/packages/pdc/package.py

```diff
@@ -14,14 +14,18 @@ class Pdc(CMakePackage):
     metadata operations to find data objects."""
 
     homepage = "https://pdc.readthedocs.io/en/latest/"
-    url      = "https://github.com/hpc-io/pdc/archive/refs/tags/0.1.tar.gz"
+    git      = "https://github.com/hpc-io/pdc.git"
```

Comment:
Leave both the `url` and `git` in to enable direct downloads but still allow git tags.